<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OTPWriteDialog</class>
 <widget class="QDialog" name="OTPWriteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="gb_otp_oper">
       <property name="title">
        <string>OTP Operation</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="rb_write_only">
          <property name="text">
           <string>Write Only</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_otp_wl">
          <property name="text">
           <string>Write and Lock</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_lock_only">
          <property name="text">
           <string>Lock Only</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gb_otp_para">
       <property name="title">
        <string>OTP Parameter</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_region">
              <property name="text">
               <string>Region:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_region"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="lb_otp_waddr">
              <property name="text">
               <string>Write Address:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_otp_waddr"/>
            </item>
            <item>
             <widget class="QLabel" name="lb_otp_wlen">
              <property name="text">
               <string>Write Length:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_otp_wlen"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="pb_OTPfile_path">
              <property name="inputMethodHints">
               <set>Qt::ImhNone</set>
              </property>
              <property name="text">
               <string>OTP file path</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_OTPfile_path"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_OK">
       <property name="text">
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Cancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>138</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
